data/hai/hai reveal 4 middle.txt

```diff
@@ -1498,7 +1498,56 @@ mission "Hai Reveal [A11-B] Permanent Fleets"
+			`	"Apologies for my brethren's behavior, Captain, but this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare that they usually would."`
+			choice
+				`	"Is politics a very important matter for the Hai?"`
+					goto politics
+				`	"What do they need to prepare for an election?"`
+			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy, than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
+			`	"Ordinarily, they would spend at least a month checking on different things and solidifying their positions before an election. A snap election has robbed them of that preparation time, and so they are stressed and concerned. Many, including myself, will have to put up with the consequences of this election for at least the next two decades before any meaningful correction can occur, should the outcome be unfortunate. Long is the timeline that a Hai merchant must consider when asked who should be supported in an election."`
+				goto ping
+			label politics
+			`	He smiles. "You must not have considered the Hai lifespan, Captain. A human might not understand the situation to have nuanced opinions until they are more than twenty years of age, and by the time they are fifty the outcome of their votes are unlikely to have a large effect on the remainder of their lives. It is comparatively easy, if things are okay, for you to simply ignore your politics in favor of other things. A Hai may live for over three hundred years, if they are quite fortunate, but will in most cases still be trying to make a livelihood halfway through that. Barring accident, stupidity, or malicious action, a Hai will typically live to see the longer term effects of their voting preferences. The more wealth a Hai has, the more significant those effects are likely to be. There are few Hai more politically aware than merchant captains, and fewer still whose opinions on the effects of change are trusted more widely.`
```

Comment:
I strongly question whether merchant captains are the most politically aware demographic, as they must spend most of their time being merchants and captains. There are many other occupations that involve investing money and researching politics. It is possible this individual is talking of his tail, though.